README.md

```diff
@@ -85,19 +86,32 @@ Setting `noReset` to `true` will leave the simulator running at the end of a tes
 The `appium-xcuitest-driver` has provisional support for iOS real devices. Not all functionality is currently supported.
 
 WebDriverAgent needs to be built with `development team` and `provisioning profile` installed on device. The easiest way
-to do this is to specify them in an xcconfig file, the path to which is passed in to the system using the
-`xcodeConfigFile` desired capability
+to do this is to specify them as desired capabilities:
+
+```
+{
+  "xcodeOrgId": "<Your Team ID>",
```

Comment:
I think either this should be "Team ID" or on line 94 it should be "Your Team ID".